app/views/includes/scripts.html

```diff
@@ -1,4 +1,4 @@
-<script src="/public/javascripts/jquery-1.11.3.js"></script>
+<script src="/public/javascripts/jquery/jquery.min.js"></script>
```

Comment:
shouldn't the min file include the version, how do we know when to update in the future without breaking open the file.

Reply:
No it doesn't :)